code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -294,7 +294,7 @@ butcher
 
 /datum/aiTask/sequence/goalbased/build/drone/precondition()
 	var/mob/living/critter/flock/F = holder.owner
-	return F?.can_afford(20)
+	return F?.can_afford(20) && (length(F?.flock?.units) > 10) //prioritise egg laying in the early game
```

Comment:
```suggestion
	return F?.can_afford(20) && F?.flock?.getComplexDroneCount() > 10) //prioritise egg laying in the early game
```

Reply:
slower, but probably worth it to be sure we're actually counting drones and not bits (could refactor `getComplexDroneCount` to be less stupid in future)

Reply:
yeah, I might make `getComplexDroneCount()` less stupid in another PR, but sure I'll commit that.